File: Bimestre1/Taller5/taller5/src/taller5/Meses.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package taller5;
import java.util.Scanner;
/**
 *
 * @author kevin
 */
public class Meses {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        //creación e inicialización de variables
        int mes;
        mes=0;
        //ingresar datos
        System.out.println("Programa para identificar el mes a través de su número");
        System.out.println("Escoja una opción del siguiente menú");
        System.out.println("1.");
        System.out.println("2.");
        System.out.println("3.");
        System.out.println("4.");
        System.out.println("5.");
        System.out.println("6.");
        System.out.println("7.");
        System.out.println("8.");
        System.out.println("9.");
        System.out.println("10.");
        System.out.println("11.");
        System.out.println("12.");
        
        System.out.print("Seleccione un numero para saber el mes: ");
        mes = leer.nextInt();
        switch(mes){
            case 1: {
                System.out.println("Enero");
            }
            break;
            case 2: {
                System.out.println("Febrero");
            }
            break;
            case 3: {
                System.out.println("Marzo");
            }
            case 4: {
                System.out.println("Abril");
            }
            break;
            case 5: {
                System.out.println("Mayo");
            }
            break;
            case 6: {
                System.out.println("Junio");
            }
            break;
            case 7 : {
                System.out.println("Julio");            
            }
            break;
            case 8 : {
                System.out.println("Agosto");
            }
            break;
            case 9:{
                System.out.println("Septiembre");
            }
            break;
            case 10: {
                System.out.println("Octubre");
            }
            break;
            case 11: {
                System.out.println("Noviembre");
            }
            break;
            case 12: {
                System.out.println("Diciembre");
            }
            break;
            default: 
                System.out.println("El número ingresado no es valido.");
            break;
        }
        
    }
}

File: Bimestre1/Taller5/PYTHON/Meses.py
# creación e inicialización de variables

mes = int(0)
# ingresar datos
print("Escoja una opción del siguiente menú")
print("1.")
print("2.")
print("3.")
print("4.")
print("5.")
print("6.")
print("7.")
print("8.")
print("9.")
print("10.")
print("11.")
print("12.")

mes = input("Seleccione un número para saber el mes: ")
if(int(mes) == 1):
    print("Enero")
if(int(mes)==2):
    print("Febrero")
if(int(mes)==3):
    print("Marzo")
if (int(mes) == 4):
    print("Abril")
if(int(mes) == 5):
    print("Mayo")
if (int(mes) == 6):
    print("Junio")
if(int(mes) == 7):
    print("Julio")
if(int(mes) == 8):
    print("Agosto")
if (int(mes) == 9):
    print("Septiembre")
if(int(mes)==10):
    print("Octubre")

if(int(mes)==11):
    print("Noviembre")

if(int(mes)==12):
    print("Diciembre")

if((int(mes)<=0)or (int(mes)>12)):
    print("El número ingresado no es valido.")